UI/css/styles.css

```diff
@@ -217,7 +221,7 @@ header ul li span.user-name {
     padding-left: 40px;
```

Comment:
Try Use the padding shorthand property with four values

Reply:
Thank you. I will work on them